### PR TITLE
Topic recognition - 4467213

### PR DESCRIPTION
**Pull request by Antoine-Le-0**
Solving third problem for recognition report
Name: Antoine Le
Student Number: 44672139

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Sufficiently detailed commit messages
- Code valid for Easy Difficulty
- Performance okay 

Other Feedback
- Breaken links in readme
- `outputs = tf.keras.layers.Conv2D(2, (1,1), activation='sigmoid')(c9)` not technically correct. You should use 1 class + sigmoid or 2 classes + softmax (less correct). Though, in this case, the results were not affected

**Comment by rexsw** [comment]
Code looks good for easy difficulty

For next time;

Your pull request should have a description of what it is and why it should be merged 

Comments such as 
    #shuffling data
     data = data.shuffle(len(images))
Add nothing to the readability of the code and should be avoided. Overall comments were well done

Code in files like driver scripts should be in a main function and in a python file.

File: recognition/44672139_topic_recognition/helper.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
+import tensorflow as tf
```
Comment (shakes76) [inline recognition/44672139_topic_recognition/helper.py:1]: Code header block should be added for all scripts/source files

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme broken links
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info. 
Notebooks have markdown sections for you to explain what's going on, so you should make use of it.
